testsuite/drivers/lkt_toolbox_driver.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
Now that it is unused, could you also remove `lkt_resolve_driver.py`?

Reply:
Done. I also removed `contrib/lkt/lkt_resolve.py`.